NSS-Spontaneous-Spoonbills/Sprint2
# Fixed testing on master

## Link to Ticket

No ticket to close 

## Description of Proposed Changes

## Steps to Test

Outline the steps to test

```sh
git fetch --all
git checkout cr_testFix
```
1. Run:

```
pyman test
```
or without the shortcut
```
python manage.py test
```
1. All 4 tests should now run and OK

## Impacted Areas in Application

* Made a WorkforceManagement/__init__.py (This NEEDS to be in the project, no more deleting or moving it!) 
*  WorkforceManagement/forms/Employee_Form.py unnecessary import was throwing a testing error 
* WorkforceManagement/test/__init__.py (Tests dont actually need to be moved into the init file, it will run everything in the file folder automatically)
* WorkforceManagement/test/test_employee_detail.py (Changed import redundancy)
* Deleted:    WorkforceManagement/test/test_views.py (Empty file)
* Deleted:    WorkforceManagement/test/test_forms.py (Empty file)

## Mentions @username

@MrErin @JacobAndrewSmith92 @binauralbeat @obscure-shadow 

## Testing


## Deploy Notes

Notes regarding deployment the contained body of work.  These should note any
db migrations, etc.

## Definition of Done

1. The project must be fully documented. This includes the following:
   1. Complete README that documents the steps to install the code, how to install any dependencies, or system configuration needed.
   1. Every class must be documented with purpose, author, and methods.
   1. Every method must be documented with purpose and argument list - which itself must contain a short purpose for each argument.
1. The project must be able to run fully, and without errors, on each teammate's system.
1. Fulfills every requirement.
1. Every line of code has been peer reviewed.
1. For projects that require unit testing, core functionality must be identified and have at least one test for each
